static/src/stylesheets/module/content/_article.scss

```diff
@@ -82,4 +82,9 @@
     .fc-container--has-show-more .treats__container {
         bottom: $gs-baseline;
     }
+
+    .fc-container--lazy-load {
+        display: block !important;
```

Comment:
Was needed here for the test but will be removed later. It's scoped to only affect divs inside the test